test/ctia/test_helpers/redis.clj

```diff
@@ -0,0 +1,8 @@
+(ns ctia.test-helpers.redis
+  (:require [ctia.test-helpers.core :as h]))
+
+(defn fixture-properties:redis-store [f]
+  ;; May be overridden with ENV variables
+  (h/with-properties ["ctia.store.redis.enabled" true
+                      "ctia.store.redis.uri" "redis://192.168.99.100:6379"]
```

Comment:
This is likely the cause of your problem, your are hardcoding the value in your text fixture.